docs/how_it_works.md

```diff
@@ -4,9 +4,7 @@ As of November 2022, and probably incomplete. This describes how the codebase cu
 
 ## The model
 
-At its heart, a graph of roads and intersections. (Roads lead between exactly two intersections -- "road segments" might be more precise.) Roads have their lanes listed from left-to-right, with a type, width, and direction. Note osm2streets doesn't model bidirectional lanes yet -- sidewalks and shared center turn lanes are either forwards or backwards right now, and something downstream interprets them in a special way. (osm2lanes has more nuance, but isn't used in osm2streets yet.)
-
-Intersections have a `ControlType` -- stop signs, traffic signals, uncontrolled, etc. This is orthogonal to `IntersectionComplexity` and `ConflictType`... TODO, narrow down valid combinations and give examples. MultiConnection vs Merge, please!
+At its heart, a graph of roads and intersections. A `Road` is a segment of road that leads between exactly two `Intersection`s. An `Intersection`'s type tells you if it represents a real-life intersection or some other kind of node in the graph. Roads have their lanes listed from left-to-right, each with a type, width, and direction. Note osm2streets doesn't model bidirectional lanes yet -- sidewalks and shared center turn lanes are either forwards or backwards right now, and something downstream interprets them in a special way. (osm2lanes has more nuance, but isn't used in osm2streets yet.)
 
```

Comment:
I'd still vote for a quick explanation here of `IntersectionType`. I think one picture/diagram with examples of all cases would do the trick. We don't need it now, but if you stumble on a small area with all examples, maybe we could just add labels

Reply:
I have added a bit more explanation. I am weary of simply reproducing good docstrings here; and of turning this doc into yet another place that describes future ideals that haven't been exercised yet, instead of providing a structural introduction that sets the reader up to understand the code as they dive in.

Reply:
Thanks for the extra explanation, and yeah, this doc is meant as a code walkthrough prep. At some point a diagram would be nice, but we don't need it now